### ISSUE TITLE
Probabilities

### ISSUE THREAD

**Issue by buerdman** [sjaehn/BSchaffl#14]
I have a feature request for this great tool: the ability to specify a probability for each step whether the MIDI note is played or not. This would be a very easy way to create constantly changing patterns.

It could be displayed graphically with faders. Either with a second fader per step or with a switch for AMP and probability. Or in another line with a small knob.

What do you think?

**Comment by sjaehn** [comment]
Nice idea. And it wouldn't be too complicated to implement it. But I do not want to overload this plugin, yet. 

And there is a more elegant way to achieve this by combining B.Schaffl with a probability-supporting MIDI sequencer. Like B.SEQuenzr.

**Comment by magnetophon** [comment]
Agreed: nice idea, but I'm not sure this plugin is the right place for it.

Have you seen https://stochas.org?
